### DISCUSSION TITLE
Oauth providers access key

### DISCUSSION THREAD

**Discussion by tigawanna** [supabase/supabase#15953] [Feature Requests]
Recently used supabase auth for a project and was frustrated by the fact that I couldn't find a way to pass in scopes and get the access token for use with their API , so I had to bring on a secondary auth server just to get the access token and the user has to sign in twice

**Comment by 26medias** [comment]
Same issue here.
I can pass scopes, but can't find a way to obtain the provider's token to call the github apis for example.
